lib/embed/embed_this_panel.rb

```diff
@@ -0,0 +1,58 @@
+module Embed
+  class EmbedThisPanel
+    def initialize(druid:, height:, width:, purl_object_title:, &block)
```

Comment:
I couldn't think of a better way to get all this information to this class.  5 args, and 5 instance variable feels like sort of a lot.